packages/ui/src/components/molecules/index.ts

```diff
@@ -10,4 +10,5 @@ export * from "./search-input";
 export * from "./status-card";
 export * from "./success-check";
 export * from "./toggle-switch";
+export * from "./token-icon/token-icon";
```

Comment:
This import looks different from the rest, i.e. `./component-name/component-name`. Is this a smell we're not being consistent with some convention we've chosen?